bundle/jetstream.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please include `JetStream part of the plgd bundle`before this one: https://github.com/plgd-dev/plgd.dev/blob/main/content/en/features/jetstream.md#deploy-jetstream-controller-for-k8s

and document how to enable it and how to change the configuration and how to configuration looks like (one from run.sh - limits.